images/macos/templates/macOS-11-anka.pkr.hcl

```diff
@@ -0,0 +1,220 @@
+variable "source_vm_name" {
+    type = string
+    default = "clean_macos_11_300gb"
+  }
+  
+  variable "source_vm_tag" {
+    type = string
+    default = "bigsur_300g"
+  }
+  
+  variable "vm_name" {
+    type = string
+    default = "71"
+  }
+  
+  variable "vcpu_count" {
+    type = string
+    default = "4"
+  }
+  
+  variable "ram_size" {
+    type = string
+    default = "6G"
+  }
+  
+  variable "build_id" {
+    type = string
+  }
+  
+  variable "image_os" {
+    type = string
+    default = "macos"
+  }
+  
+  source "veertu-anka-vm-clone" "template" {
+    vm_name = "${var.vm_name}"
+    source_vm_name = "${var.source_vm_name}"
+    source_vm_tag = "${var.source_vm_tag}"
+    vcpu_count = "${var.vcpu_count}"
+    ram_size = "${var.ram_size}"
+    stop_vm =  "true"
+  }
+  
+  build {
+    sources = [
+      "source.veertu-anka-vm-clone.template",
+    ]
+    provisioner "shell" {
+      inline = [
+        "mkdir ~/image-generation"
+      ]
+    }
+    provisioner "file" {
+      destination = "image-generation/"
+      source      = "./provision/assets"
+    }
+    provisioner "file" {
+      destination = "image-generation/"
+      source      = "./tests"
+    }
+    provisioner "file" {
+      destination = "image-generation/"
+      source      = "./software-report"
+    }
+    provisioner "file" {
+      destination = "image-generation/"
+      source      = "./helpers"
+    }
```

Comment:
I think we can unite all these provisioners into the single one.
`file` provisioner supports `sources` argument: https://www.packer.io/docs/provisioners/file